frontend/src/features/books/BookDetails.jsx

```diff
@@ -1,27 +1,22 @@
 import React, { useEffect, useState } from 'react'
-import { useParams, Link } from 'react-router-dom'
+import { useParams, Link, useNavigate } from 'react-router-dom'
 import { API_URL } from '../../constants'
 
 function BookDetails() {
-  const [title, setTitle] = useState(null)
-  const [author, setAuthor] = useState(null)
-  const [description, setDescription] = useState(null)
+  const [book, setBook] = useState(null)
   const { id } = useParams()
+  const navigate = useNavigate()
 
   useEffect(() => {
     const fetchCurrentBook = async () => {
       try {
         const response = await fetch(`${API_URL}/${id}`)
```

Comment:
@nshelokar-vf, did we handle the case where `id` is not present?

Reply:
Currently not, But i will add it.